data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
```

Comment:
Maybe specify it as default for subtypes then? Vast majority of fruit is volume 1 or volume 1 and stack size 4 or 8.

Per-item volume 1 is an overkill for sets of nearly identical objects (ie. fruit) and per-category defaults are less ugly than hardcoded defaults.

This may change if the "volume to ml" PR lands (and obsoletes stack size), but until then, it's a reasonable assumption that fruit has 1 volume.